indexer/services/comlink/src/controllers/api/v4/compliance-controller.ts

```diff
@@ -1,33 +1,24 @@
 import { logger, stats, TooManyRequestsError } from '@dydxprotocol-indexer/base';
 import { ComplianceClientResponse, INDEXER_COMPLIANCE_BLOCKED_PAYLOAD } from '@dydxprotocol-indexer/compliance';
 import { ComplianceDataFromDatabase, ComplianceTable } from '@dydxprotocol-indexer/postgres';
-import express from 'express';
-import { checkSchema, matchedData } from 'express-validator';
 import { DateTime } from 'luxon';
 import {
-  Controller, Get, Query, Route,
+  Controller, Get, Query,
 } from 'tsoa';
 
 import {
-  getReqRateLimiter,
   screenProviderLimiter,
   screenProviderGlobalLimiter,
 } from '../../../caches/rate-limiters';
 import config from '../../../config';
 import { complianceProvider } from '../../../helpers/compliance/compliance-clients';
-import { create4xxResponse, handleControllerError } from '../../../lib/helpers';
-import { rateLimiterMiddleware } from '../../../lib/rate-limit';
-import { getIpAddr } from '../../../lib/utils';
-import { handleValidationErrors } from '../../../request-helpers/error-handler';
-import ExportResponseCodeStats from '../../../request-helpers/export-response-code-stats';
-import { ComplianceRequest, ComplianceResponse } from '../../../types';
+import {ComplianceResponse } from '../../../types';
 
-const router: express.Router = express.Router();
 const controllerName: string = 'compliance-controller';
 const UNCACHED_QUERY_POINTS: number = 1;
 const GLOBAL_RATE_LIMIT_KEY: string = 'screenQueryProviderGlobal';
 
-@Route('screen')
+
```

Comment:
can you move the remaining helper functions in this file to compliance-v2-controller.ts?